src/lib.rs

```diff
@@ -423,16 +458,173 @@ pub trait PolynomialCommitment<F: Field>: Sized {
 
         Ok(true)
     }
+
+    /// open but with individual challenges
+    /// By default, we downgrade them to only use the first individual opening challenges
```

Comment:
What does this mean? Does it mean that there's one challenge per polynomial? If so, what does it mean to `downgrade them to only use the first individual opening challenges`?

Reply:
Ah I understand now.

Maybe we should rephrase this to say that `By default, this method invokes `Self::open` with `opening_challenge = opening_challenges[0]?`.

Reply:
Actually, I think we should reverse the default implementation here, if possible: `open` should call `open_individual_opening_challenges` with `opening_challenges = |pow| opening_challenge.pow(&[pow]);`, i.e. the same impl as in `MarlinKZG`.

Reply:
Fixed in the incoming commit.